tests/sentry/integrations/slack/webhooks/actions/test_status.py

```diff
@@ -173,140 +226,130 @@ def test_ignore_issue(self):
-        assert event.group is not None
-        resp = self.post_webhook(
-            action_data=[status_action],
-            original_message=self.original_message,
-            type="interactive_message",
-            callback_id=json.dumps({"issue": event.group.id}),
-        )
-        self.group = Group.objects.get(id=event.group.id)
+    @responses.activate
+    def test_archive_issue_until_escalating_block_kit_through_unfurl(self):
+        original_message = self.get_original_message_block_kit(self.group.id)
+        payload_data = self.get_block_kit_unfurl_data(original_message["blocks"])
+        self.archive_issue_block_kit(original_message, "ignored:until_escalating", payload_data)
 
-        assert resp.status_code == 200, resp.content
+        self.group = Group.objects.get(id=self.group.id)
         assert self.group.get_status() == GroupStatus.IGNORED
         assert self.group.substatus == GroupSubStatus.UNTIL_ESCALATING
 
-        expect_status = f"Identity not found.\n*Issue archived by <@{self.external_id}>*"
-        assert resp.data["attachments"][0]["text"] == expect_status
+        update_data = json.loads(responses.calls[1].request.body)
 
-        with self.feature("organizations:slack-block-kit"):
-            # test backwards compatibility
-            resp = self.post_webhook(
-                action_data=[status_action],
-                original_message=self.original_message,
-                type="interactive_message",
-                callback_id=json.dumps({"issue": event.group.id}),
-            )
-            self.group = Group.objects.get(id=event.group.id)
+        expect_status = f"*Issue archived by <@{self.external_id}>*"
+        assert update_data["blocks"][0]["text"]["text"].endswith(expect_status)
 
-            assert resp.status_code == 200, resp.content
-            assert self.group.get_status() == GroupStatus.IGNORED
-            assert self.group.substatus == GroupSubStatus.UNTIL_ESCALATING
-            # XXX(CEO): it's kind of odd to code format this but would be tricky to avoid
-            expect_status = f"```Identity not found.```\n*Issue archived by <@{self.external_id}>*"
-            assert resp.data["blocks"][0]["text"]["text"].endswith(expect_status)
+    @responses.activate
+    def test_archive_issue_until_condition_met_block_kit(self):
+        original_message = self.get_original_message_block_kit(self.group.id)
+        self.archive_issue_block_kit(original_message, "ignored:until_condition_met:10")
 
-    def test_archive_issue_block_kit(self):
-        event = self.store_event(
-            data=self.event_data,
-            project_id=self.project.id,
+        self.group = Group.objects.get(id=self.group.id)
+        assert self.group.get_status() == GroupStatus.IGNORED
+        assert (
+            self.group.substatus == GroupSubStatus.UNTIL_CONDITION_MET
+        )  # TODO: see if we can get the ignoreCount?
```

Comment:
Is that the x of "archive until it happens x times"?

Reply:
yes i asked around and i think i need to query groupsnooze